joystick_driver:
  ros__parameters:
    absoluteMaxVel: 1.0
    maxVel: 1.6
    maxVelIncr: 0.1
    leftJoyAxis: 1
    rightJoyAxis: 3
    leftInverted: True
    rightInverted: True